### PR TITLE
Performance Improvements for Audits

### PR DESCRIPTION
**Pull request by dmzimmerman**
I completely overhauled how the `cvr_audit_info` is tracked, and used, in audits. This eliminates a _whole lot_ of unnecessary database queries. I also restored the functionality of the `ballot_under_audit_id` to correctly indicate the actual ballot that should be under audit in the sequence we are displaying them, to go with the client changes that have not yet been merged.

### CODE REVIEW

File: server/eclipse-project/src/main/java/us/freeandfair/corla/query/CVRAuditInfoQueries.java [hunk 1]
```diff
@@ -1,344 +0,0 @@
-/*
```
Comment (kiniry) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/query/CVRAuditInfoQueries.java]: Begone!

File: server/eclipse-project/pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>us.freeandfair.production</groupId>
 	<artifactId>colorado_rla</artifactId>
-	<version>1.0.5</version>
+	<version>1.1.0-dev</version>
```
[the remark this answers is not in the document]
Comment (dmzimmerman) [reply server/eclipse-project/pom.xml]: Yup.

[the remark this answers is not in the document]
Comment (dmzimmerman) [reply server/eclipse-project/pom.xml]: And the massive efficiency gains.

**Comment by nealmcb** [comment]
Nice work!  I see about a 70% increase in CVR import parsing speed with this PR.

But there are major implications for `rla_export` queries.

**Pull request merged**